app-web/qbittorrent/02-nox/prepare

```diff
@@ -0,0 +1,4 @@
+abinfo "Cleaning ..."
+make distclean
+abinfo "Running configure script to generate conf.pri ..."
+"$SRCDIR"/configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var --enable-stacktrace --disable-gui --enable-systemd --enable-webui
```

Comment:
Wouldn't there be file conflict with the standard version?

Reply:
Nope, it's intended to be not overlapping.